src/index.tsx

```diff
@@ -1,8 +1,20 @@
 import ReactDOM from 'react-dom';
 
 import { App } from './App';
+import { UsersProvider } from './context/UsersContext';
+import { PostsProvider } from './context/PostsContext';
+import { ErrorProvider } from './context/ErrorContext';
+import { CommentsProvider } from './context/CommentsContext';
 
 ReactDOM.render(
-  <App />,
+  <UsersProvider>
+    <PostsProvider>
+      <CommentsProvider>
+        <ErrorProvider>
+          <App />
+        </ErrorProvider>
+      </CommentsProvider>
+    </PostsProvider>
+  </UsersProvider>,
```

Comment:
you can create a component that will wrap the App with all providers

Reply:
That makes sense - you should do it next time

Reply:
That makes sense - you should do it next time